base/docs/basedocs.jl

```diff
@@ -2106,10 +2106,16 @@ Creates a null pointer to type `T`.
 """
 Ptr{T}()
 
+"""
+    `plus` is not used in Julia.
+Use `+` for adding numbers together, but note, `*` is used in Julia to concatenate strings.
+"""
+kw"plus"
+
 """
     +(x, y...)
 
-Addition operator. `x+y+z+...` calls this function with all arguments, i.e. `+(x, y, z, ...)`.
+Addition operator. `x+y+z+...` calls this function with all arguments, i.e. `+(x, y, z, ...)`. Note, to concatenate strings in Julia, use the `*` operator.
```

Comment:
This does seem like a useful hint, because `+` for string concatenation is used in a majority of languages, most notably Python, which is probably the most common language Julia users switch from. The others seem to only be used in more obscure languages, so I'm a bit more skeptical about them. It also seems like the discussion in #37469 was generally against adding `plus`.

Reply:
SQL isn't really an obscure language, if you mean that, but `||` is unusual and I know it only used there. That however is just an addition to an already existing help item (for completeness, more or less, I know ++ is used in some language, I just wanted to cover the popular SQL).